# Goal
geometry_msgs/Pose2D goal_pos
---
# Result
bool status
---
# Feedback
geometry_msgs/Pose2D current_pos